x-pack/test/functional/apps/maps/index.js

```diff
@@ -9,14 +9,38 @@ export default function ({ loadTestFile, getService }) {
   const kibanaServer = getService('kibanaServer');
   const esArchiver = getService('esArchiver');
   const browser = getService('browser');
+  const log = getService('log');
+  const supertest = getService('supertest');
 
   describe('maps app', function () {
     this.tags(['skipFirefox']);
 
     before(async () => {
       await esArchiver.loadIfNeeded('x-pack/test/functional/es_archives/logstash_functional');
+      await kibanaServer.importExport.load(
+        'x-pack/test/functional/fixtures/kbn_archiver/maps.json'
+      );
+      //Find the missing references manually, use the below API to delete it after successful import.
```

Comment:
Nit: mind changing the comment to include reason why index pattern saved objects have to be delete?

How about something like 
```
// Functional tests verify behavior when referenced index pattern saved objects can not be found.
// However, saved object import fails when reference saved objects can not be found.
// To prevent import errors, index pattern saved object references exist during import 
// but are then deleted afterwards to enable testing of missing reference index pattern saved objects..
```

Reply:
sure, merged this PR too quickly. Addressed it in a separate PR #103892